### PR TITLE
Maintenance of LuaJIT integration CI

### PR DESCRIPTION
**Pull request by igormunkin**
Small changes to remove the old code from LuaJIT integration CI:
* Removed the obsolete `inputs.host` parameter
* Enabled `CMAKE_EXTRA_PARAMS` for .test.mk (since #8026 has been fixed)

---

P.S. LuaJIT CI is [green](https://github.com/tarantool/luajit/commit/52ff4c3) as a result of this patch.

### CODE REVIEW

File: .test.mk [hunk 1]
```diff
@@ -275,10 +275,10 @@ test-coverity: build-coverity
 # LuaJIT integration testing #
 ##############################
 
-test-luajit-linux-x86_64: build run-luajit-test run-test
+test-luajit-Linux-x86_64: build run-luajit-test run-test
```
Comment (ylobankov) [inline .test.mk:278]: Actually, I like this naming :) Can we keep it further?

File: .github/workflows/luajit-integration.yml [hunk 2]
```diff
@@ -2,25 +2,18 @@
...
+        required: true
         type: string
-        # XXX: Temporary ugly hack: use 'self-hosted' as a default
-        # value to fool GitHub Actions machinery choosing the
-        # appropriate runner for the workflow (since the empty
-        # string is not a valid parameter).
-        default: self-hosted
       # FIXME: Remove this inputs entry when transition to
       # CMAKE_EXTRA_PARAMS usage is finished.
       buildtype:
```
[the remark this answers is not in the document]
Comment (igormunkin) [reply .github/workflows/luajit-integration.yml:19]: Unfortunately, not yet: `buildtype` and `GC64` are still being used in the long-term LuaJIT branches. They will be removed when this PR is merged into the trunk.

File: .test.mk (hunk 1 above)
Comment (igormunkin) [reply .test.mk:278]: We use this values within LuaJIT matrix and I, actually, don't want to make additional tweaks to obtain the values for the current naming.

I suggest to try the new naming and return to this question, if we face issues with it (either Makefile targets mislookup or its complex maintenance).

**Review by ylobankov — APPROVED** [review]
LGTM

**Comment by igormunkin** [comment]
@ylobankov, thanks for your review!

> LGTM

Added your tag:
| Reviewed-by: Yaroslav Lobankov <y.lobankov@tarantool.org>

**Pull request merged**